KubeArmor/core/karmorprobedata.go

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2022 Authors of KubeArmor
+
+package core
+
+import (
+	"encoding/json"
+	"log"
+	"os"
+
+	cfg "github.com/kubearmor/KubeArmor/KubeArmor/config"
+)
+
+type KarmorData struct {
+	OSImage             string
+	KernelVersion       string
+	KubeletVersion      string
+	ContainerRuntime    string
+	ActiveLSM           string
+	KernelHeaderPresent bool
+	HostSecurity        bool
+	ContainerSecurity   bool
+	KubeArmorPosture    KubeArmorPostures
```

Comment:
```suggestion
	ContainerDefaultPosture    tp.DefaultPosture
	HostDefaultPosture    tp.DefaultPosture
```
Ref https://pkg.go.dev/github.com/kubearmor/KubeArmor/KubeArmor/types#DefaultPosture